includes/admin/class.llms.admin.assets.php

```diff
@@ -285,7 +285,7 @@ public function admin_print_scripts() {
 			echo "<script>window.llms.formLocations = JSON.parse( '" . wp_json_encode( wp_slash( LLMS_Forms::instance()->get_locations() ) ) . "' );</script>";
 		}
 
-		if ( ! empty( $screen->is_block_editor ) ) {
+		if ( ! empty( $screen->is_block_editor ) || 'customizer' === $screen ) {
```

Comment:
I didn't test this but I'm guessing this isn't working like https://github.com/gocodebox/lifterlms-blocks/pull/110